api/lib/application/campaign-participations/campaign-participation-controller.js

```diff
@@ -1,4 +1,4 @@
-const performanceTool = require('../../infrastructure/performance-tools');
+const monitoringTool = require('../../infrastructure/monitoring-tools');
```

Comment:
Pour garder une cohérence entre le nom du module et le nom sous lequel il est importé :
```suggestion
const monitoringTools = require('../../infrastructure/monitoring-tools');
```